Classes/Controller/StyleguideController.php

```diff
@@ -95,11 +97,31 @@ public function showAction(array $arguments = [])
             '>='
         );
 
+        if ($this->styleguideConfigurationManager->isFeatureEnabled('CodeQuality') && class_exists(CodeQualityService::class)) {
+            $showQualityIssues = true;
+
+            // Initialize code quality service
+            $configurationService = new ConfigurationService;
+            $configuration = $configurationService->getFinalConfiguration(false, $component->getCodeQualityConfiguration());
```

Comment:
If no configuration file is found:
`$component->getCodeQualityConfiguration()` returns `null`

But the check for no configuration file in fluid-conponents-linter `ConfigurationService` is `$configurationFile !== false`

So we pass `null` as second parameter to the function which expects `false` as indicator for no configuration file. This results in an error.

The crux of non typed parameters.

**Reproduce:** usage without any `.fclint.json` in project.